### PR TITLE
Fixes the issue realted to adding/ updating events tags not been saved and making form unsubmissive

### LINKED ISSUE #10197: [BUG] While adding/ updating events tags are not been saved and making form unsubmissive
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

production

### Version of BioDrop (for example "v1.2.3")

2.99.10

### Description

If you are creating / updating events and want to add tags, the `Save` button gets disabled and in case tags are removed, it remains disabled

### Steps to Reproduce

1. Go to [Events Page](https://www.biodrop.io/account/manage/event) page.
2. Enter all the required details without adding any tags.
3. **`Save`** button remains active 
4. Now try to add tags like `javascript` and enter it will be added in the below box
5. Now the **`Save`** button gets disabled. 
6. If you remove the added tags the button remains disabled.

- This is happening in both cases i.e. while adding an event and also while updating an event.
both

### Screenshots

Without tags
With tags

### Do you want to work on this issue?

Yes

### If "yes" to above, please explain how you would technically implement this

The issue is that the `Save` button is not getting its validated value reset in case tags are added or removed. This is most probably a form validation issue.

### PR DESCRIPTION
**Pull request by amanpreet-dev**
## Fixes Issue
Closes #10197 

## Changes proposed

- `Description` field was made required as per the `Event` schema
- Commented out `setIsDisabled(true)` in the `handleSubmit()` function as this was making the form disabled after inputting the tags.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Creating a new Event
Creating an Event

Updating Event
Updating an Event

## Note to reviewers

I have added and updated some sample events locally to test the application flow and it was working fine.

### CODE REVIEW

File: pages/account/manage/event/[[...data]].js [hunk 1]
```diff
@@ -106,8 +106,8 @@ export default function ManageEvent({ BASE_URL, event }) {
 
   const handleSubmit = async (e) => {
     e.preventDefault();
-    setIsDisabled(true);
-
+    // setIsDisabled(true);
```
Comment (ThomasCode92) [inline pages/account/manage/event/[[...data]].js:109]: If this code isn't necessary, feel free to remove it.

**Review by luismr00 — COMMENTED** [review]
I tested the new change and everything works. However, the button does not get disabled anymore whenever you are missing a field. This probably does not matter because even if you hit the save button, it will throw an alert if you are missing a field. I would ask another maintainer if they are ok with this, or if they still want the button to become disabled whenever input fields are missing.

**Comment by amanpreet-dev** [comment]
Can I get an update on this PR? if it is still having an issue then please do let me know.

**Comment by eddiejaoude** [comment]
Thanks @amanpreet-dev for the PR.

Good spot @luismr00 thank you for testing 👍

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍  I will merge and tweak accordingly

**Pull request merged**